packages/flutter_list_bloc/example/lib/showcases/page_view_bloc_builder_showcase.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:example/utils/api_emulator.dart';
+import 'package:example/widgets/empty_indicator.dart';
+import 'package:example/widgets/item_tile.dart';
+import 'package:example/widgets/loading_data_indicator.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_list_bloc/flutter_list_bloc.dart';
+import 'package:list_bloc/list_bloc.dart';
+
+class PageViewBlocBuilderShowcase extends StatelessWidget {
+  const PageViewBlocBuilderShowcase({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final cubit = PaginatedCubit<String, int>(
```

Comment:
@maksganzol Won't `BlocProvider` work here? Otherwise this cubit may cause memory leak, it is not disposed properly.